CenterForOpenScience/cas-overlay
# [SVCS-579] Enhance CAS Login Pages for Institution Login

### Ticket

https://openscience.atlassian.net/browse/SVCS-579

### Purpose

* Enhance institution login by making a button (similar to ORCiD) to replace the link.

### Side Effects

* Every customized pages in CAS needs an update.

### Changes

* Default Login Page

![default-login](https://user-images.githubusercontent.com/3750414/36050603-25e01bbe-0db5-11e8-897c-2ce65eca9b52.png)

* Institution Login Page

![inst-login](https://user-images.githubusercontent.com/3750414/36050610-2ca59276-0db5-11e8-8b4b-06032bc9c44b.png)

* Two-factor Auth Page

![2fa-login](https://user-images.githubusercontent.com/3750414/36050624-368ac5cc-0db5-11e8-9d8c-74f48c874f16.png)

* OAuth Page

![oauth-authorize](https://user-images.githubusercontent.com/3750414/36050691-7cc6f844-0db5-11e8-89a5-ca30a32ef0b8.png)

Please refer to the ticket for more pages (errors, internal, etc.)

### QA Notes

Test the following pages works as expected (links, buttons, language ...)

* Login Pages: default, institution, two-factor, oauth authorize page
* Login Failure Pages: account disabled, account not confirmed, remote login failed, oauth failed
* If you don't see the changes, you may need to delete the cache and do a hard reload

Note: I will test internal pages which is not available for users.

### Deployment Notes

No. Just deploy.
